src/client/src/rt-storybook/Story/Story.tsx

```diff
@@ -66,10 +60,10 @@ const StyledStory = styled('div')`
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  background-color: ${({ theme: { background } }) => background.backgroundPrimary};
-  color: ${({ theme }) => theme.text.primary};
-  transition: background-color ${({ theme }) => theme.animationSpeed.normal}ms,
-    color ${({ theme }) => theme.animationSpeed.normal}ms;
+  background-color: ${p => p.theme.shell.backgroundColor};
+  color: ${p => p.theme.shell.textColor};
+  transition: background-color ${p => p.theme.motion.duration}ms ${p => p.theme.motion.easing},
+    color ${p => p.theme.motion.duration}ms ${p => p.theme.motion.easing};
```

Comment:
- Change `p`  to `props` for readability
- Destructure theme if you want

Reply:
more characters, more problems